lib/sedna/service/server/knowledgeBase/server.py

```diff
@@ -143,13 +142,9 @@ def update_status(self, data: KBUpdateResult = Body(...)):
                 continue
             new_task_group.append(task_group)
         task_info["task_groups"] = new_task_group
-        self.latest += 1
-
-        _index_path = FileOps.join_path(self.save_dir,
-                                        f"kb_index_{self.latest}.pkl")
-        joblib.dump(task_info, _index_path)
-        res = f"/file/download?files=kb_index_{self.latest}.pkl&name=index.pkl"
-        return res
+        _index_path = FileOps.join_path(self.save_dir, self.kb_index)
+        FileOps.dump(task_info, _index_path)
+        return f"/file/download?files={self.kb_index}&name={self.kb_index}"
 
     def update(self, task: UploadFile = File(...)):
```

Comment:
The update function is important in the knowledge base. Comments are needed to state what is done in this function and what are its input and output.

Reply:
As above, we will complete in the PR of document supplement

Reply:
Glad to hear that the comment will be followed up. It would be great if we have related issues in the community.

Reply:
Please follow #85 and #150 to facilitate  the docs supplement for `lifelonglearning` and `sedna sdk`